extensions/jdbc/jdbc-db2/deployment/pom.xml

```diff
@@ -48,6 +48,11 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.transformer</groupId>
+            <artifactId>org.eclipse.transformer</artifactId>
+            <version>0.5.0</version>
+        </dependency>
```

Comment:
Move this to our BOM?

Reply:
I think I would rather not enforce it for now. We can revisit if we have more modules using it but I would rather let projects use the version they want given we only use a small capability of it.

Reply:
Right, I was in doubt about this and leaning for the less disruptive approach keeping it here. Happy to revisit when there's any need.